lib/tower_cli/resources/job.py

```diff
@@ -209,6 +209,14 @@ def launch(self, job_template=None, monitor=False, timeout=None,
         if supports_job_template_launch and not use_job_endpoint:
             job_id = job_started.json()['job']
 
+        # If 'ignored_fields' is present in the returning json, display it
+        # in verbose mode.
+        if 'ignored_fields' in job_started.json():
```

Comment:
This field will be present in the response even if no fields are ignored. As a specific example, if no variables are rejected, it will look like this:

```json
    "ignored_fields": {
        "extra_vars": {}
    },
```

It's these cases that the `in` logic will often miss. I realize that this might be very tricky for you to correctly check for.

We also can't _count_ on the fields being there, because it really won't be present in 2.4.5, which we still have to support. Let me know if you have any questions on these stipulations.